### PR TITLE
AWS Kommander Docs 

### PR DESCRIPTION
**Pull request by cmastr**
Signed-off-by: Casie Oxford <coxford@d2iq.com>

http://docs-d2iq-com-pr-3992.s3-website-us-west-2.amazonaws.com/ 

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-79405 

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by dkoshkin — APPROVED** [review]

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/kommander/2.1/download/index.md [hunk 1]
```diff
@@ -2,18 +2,32 @@
...
+
+## Download from the AWS Marketplace
+
+Follow the instructions on AWS console to download the container image. 
+
+After downloading the image, run the following command to copy the binaries onto your host.
 
-If you have problems downloading Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.</p>
+    ```sh
+    $ docker run -it --rm -u $(id -u):$(id -g) -v $(pwd):/dkp $CONTAINER_IMAGES
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/download/index.md:28]:
```suggestion
    docker run -it --rm -u $(id -u):$(id -g) -v $(pwd):/dkp $CONTAINER_IMAGES
```

File: pages/dkp/kommander/2.1/download/index.md [hunk 2]
```diff
@@ -2,18 +2,32 @@
...
+    dkp and kommander binaries placed in the local directory, to run:
+    ./dkp --help
+    ./kommander --help
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/download/index.md:31]:
```suggestion
```
was this meant to be here?

File: pages/dkp/kommander/2.1/download/index.md [hunk 3]
```diff
@@ -2,18 +2,32 @@
...
+        ```
+You should now see two binaries: `dkp` and `kommander` in your working directory. Follow the [Kommander installation instructions](../install/) using these binaries. If you have problems downloading Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.</p>
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/download/index.md:33]:
```suggestion

You should now see two binaries: `dkp` and `kommander` in your working directory. Follow the [Kommander installation instructions](../install/) using these binaries. If you have problems downloading Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

File: pages/dkp/kommander/2.1/download/index.md (hunk 2 above)
Comment (dkoshkin) [reply pages/dkp/kommander/2.1/download/index.md:31]: Thats the output of running the docker run command

**Review by JohnB64-JBV — APPROVED** [review]

File: pages/dkp/kommander/2.1/install/networked/index.md [hunk 4]
```diff
@@ -9,9 +9,11 @@ beta: false
 
 ## Prerequisites
 
-The `kommander` CLI installs Kommander. You must install Kommander before executing these commands. Ensure you have the version of the CLI that matches the Kommander version you want to install. Refer to the [**DOWNLOAD**][download] page to download and install Kommander.
+Before you begin using Kommander, you must:
 
-The `cert-manager` must be installed prior to installing Kommander.
+- Configure a Konvoy cluster using the [Konvoy Install](/pages/dkp/konvoy/2.1/choose-infrastructure/) instructions.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/install/networked/index.md:14]:
```suggestion
- Configure a Konvoy cluster using the [Konvoy Install](/dkp/konvoy/2.1/choose-infrastructure/) instructions.
```

File: pages/dkp/kommander/2.1/install/networked/index.md [hunk 5]
```diff
@@ -9,9 +9,11 @@ beta: false
...
+- Install `cert-manager` prior to installing Kommander.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/install/networked/index.md:15]: Now that we're setting Konvoy as a prereq - we can add this bit. I'm 99.9% sure on this, but can you check me @dkoshkin @gracedo ?

```suggestion
- Install `cert-manager` prior to installing Kommander. (Konvoy does this as part of its install process).
```

File: pages/dkp/kommander/2.1/licensing/purchase/index.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+## Purchase a License
+
+To obtain a valid Kommander license:
+
+-   Contact your sales representative at <sales@d2iq.com> to purchase one or more licenses.
+-   Purchase a license via the [AWS Marketplace](https://aws.amazon.com/marketplace/). After purchase, your license information (ARN) is accessible in the AWS License Manager console.
+
+After purchase, [download](../../../../../download/) the binary files.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/licensing/purchase/index.md:17]:
```suggestion
After purchase, [download](../../download/) the binary files.
```

File: pages/dkp/kommander/2.1/licensing/purchase/index.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+layout: layout.pug
+navigationTitle: Purchase a Kommander license
+title: Purchase
+excerpt: Obtain licenses for Kommander clusters
+menuWeight: 0
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/licensing/purchase/index.md:6]:
```suggestion
menuWeight: 20
```

File: pages/dkp/kommander/2.1/licensing/remove/index.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
+---
+layout: layout.pug
+navigationTitle: Remove a Kommander license
+title: Remove a Kommander license
+excerpt: Remove licenses for Kommander clusters
+menuWeight: 0
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/licensing/remove/index.md:6]:
```suggestion
menuWeight: 30
```

File: pages/dkp/kommander/2.1/licensing/purchase/index.md (hunk 7 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/licensing/purchase/index.md:6]: So, don't know if this is right, but purchase and remove were both set to `0`.

So, I was thinking that it should be 
Licensing
- Add a license (most important)
- Purchase a license
- Remove a license 

But, feel free to call me on this

**Review by msdolbey — APPROVED** [review]
👍 
Looks good - some link things, and a few nits, but that's all for me.

Take my money

File: pages/dkp/kommander/2.1/licensing/purchase/index.md (hunk 7 above)
Comment (cmastr) [reply pages/dkp/kommander/2.1/licensing/purchase/index.md:6]: yeah that''s a good call Matt. Thank you!

**Pull request merged**